ios/Classes/GigyaSdkWrapper.swift

```diff
@@ -190,6 +190,29 @@ public class GigyaSdkWrapper<T: GigyaAccountProtocol> :GigyaInstanceProtocol {
         })
     }
     
+    /**
+     Forgot password
+     */
+    func forgotPassword(arguments: [String: Any], result: @escaping FlutterResult) {
+        guard let loginId = arguments["loginId"] as? String else {
+            result(FlutterError(code: PluginErrors.missingParameterError, message: PluginErrors.missingParameterMessage, details: nil))
+            return
+        }
+        sdk?.forgotPassword(loginId: loginId, completion: { gigyaResponse in
+            switch gigyaResponse {
+            case .success( _):
+                result(nil)
```

Comment:
It is required to deliver the response to the client even if the flow is not resolvable.

Reply:
Done